clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1101,6 +1101,9 @@ public void assign(Collection<TopicPartition> partitions) {
             long remaining = timeout;
             do {
                 Map<TopicPartition, List<ConsumerRecord<K, V>>> records = pollOnce(remaining);
+
+                System.out.println("Polled records: " + records);
```

Comment:
maybe trace level logging instead?

Reply:
he said "The debugging lines will be removed when I remove the WIP tag." :)